### PR TITLE
Added host inventory option to zabbix_host_facts module

### PR DESCRIPTION
**Pull request by sky-joker**
##### SUMMARY
This pull request adds an option to get the host inventory to zabbix_host_facts.  
Useful for automation based on host inventory info.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME

zabbix_host_facts.py

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/modules/monitoring/zabbix/zabbix_host_facts.py [hunk 1]
```diff
@@ -54,6 +54,13 @@
             - Remove duplicate host from host result
         type: bool
         default: yes
+    host_inventory:
+        description:
+            - Specify host inventory name.
```
Comment (D3DeFi) [inline lib/ansible/modules/monitoring/zabbix/zabbix_host_facts.py:59]: Current description is not very clear to me, I assume it was meant to be something like this: 
```yaml
- List of host inventory keys to display in result
- Whole host inventory is retrieved if keys are not specified
```

**Review by D3DeFi — CHANGES_REQUESTED** [review]
Thank you for this PR @sky-joker. As per [Ansible documentation](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_general.html?highlight=devel#contributing-back-to-ansible):
"f you would like to contribute to the main Ansible repository by adding a new feature or fixing a bug, create a fork of the Ansible repository and develop against a new feature branch using the devel branch as a starting point. When you you have a good working code change, you can submit a pull request to the Ansible repository by selecting your feature branch as a source and the Ansible devel branch as a target."

Please close this PR and open a new one from different source branch than "devel" (e.g. use something like "zabbix-host-facts-inventory").

Otherwise I do like your code and will vote for it to be merged.

File: lib/ansible/modules/monitoring/zabbix/zabbix_host_facts.py [hunk 2]
```diff
@@ -70,6 +77,21 @@
     timeout: 10
     exact_match: no
     remove_duplicate: yes
+
+- name: Specify host inventory name and get host info
```
Comment (D3DeFi) [inline lib/ansible/modules/monitoring/zabbix/zabbix_host_facts.py:81]: Same as above, description is not very clear:
```yaml
- name: Reduce host inventory information to provided keys
```

**Comment by sky-joker** [comment]
@D3DeFi 

Thank you for the review!

> Please close this PR and open a new one from different source branch than "devel" (e.g. use something like "zabbix-host-facts-inventory").
Thank you for letting me know.
I close the pull request and change the branch name.

File: lib/ansible/modules/monitoring/zabbix/zabbix_host_facts.py (hunk 1 above)
Comment (sky-joker) [reply lib/ansible/modules/monitoring/zabbix/zabbix_host_facts.py:59]: Thank you.
I will modify the description.

File: lib/ansible/modules/monitoring/zabbix/zabbix_host_facts.py (hunk 2 above)
Comment (sky-joker) [reply lib/ansible/modules/monitoring/zabbix/zabbix_host_facts.py:81]: Thank you.
I will modify task name of examples.

**Pull request closed without merging**